README.md

```diff
@@ -0,0 +1,42 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+My first project at microverse
+
+
+## Built With
+
+- Css and Html
+- vscode
+- git and github
+
+
+
+## Authors
+
+👤 **Wolf**
+
+- GitHub: [@githubhandle](https://github.com/Wolfmikerson)
+- Twitter: [@twitterhandle](https://twitter.com/wolfmikerson)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/wolf-m-jacques-27a38612a/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse community
+- Github community
+- Youtube community
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] to keep your readme professional ,  kindly fix your MIT License link it's not working